src/locale/es_ES.ts

```diff
@@ -23,6 +23,7 @@ export default {
     },
     GsStyle: {
         addRuleBtnText: 'Añadir regla',
+        cloneRulesBtnText: 'Duplicadas reglas',
```

Comment:
Very minor: I think it should be `Duplicar reglas`.

Reply:
OK, my Spanish is a bit rusty and this is what an online translator gave me :laughing: